### DISCUSSION TITLE
[BUG?] Minimised windows are not differentiated in Win10 style taskbar.

### DISCUSSION THREAD

**Discussion by coworo** [valinet/ExplorerPatcher#3024] [General]
Not sure if this is an oversight or a bug.

Demonstration : One of these paints is active, one of them is inactive with window visible, one of them is minimized. There is no visual difference between inactive and minimized windows.

![image](https://github.com/valinet/ExplorerPatcher/assets/98850730/faf67b7b-bcb2-4a19-b49e-150e31663880)

**Comment by Amrsatrio** [comment]
It has always been like this since Windows 7. There's no distinction between them.
